query/gizmo/gizmo_test.go

```diff
@@ -736,3 +736,70 @@ func issue718Nodes() []string {
 	}
 	return nodes
 }
+
+var testQueriesMultiGraph = []struct {
+	message string
+	data    []quad.Quad
+	query   string
+	limit   int
+	tag     string
+	expect  []string
+	err     bool // TODO(dennwc): define error types for Gizmo and handle them
+}{
+	{
+		message: "issue #758. Verify saveOpt respects label context",
+		query: `
+			g.V("<greg>").LabelContext("<smart_graph>").SaveOpt("<status>", "statusTag").All()
+		`,
+		tag:    "statusTag",
+		expect: []string{"smart_person"},
+	},
+	{
+		message: "issue #758. Verify saveR respects label context.",
+		query: `
+			g.V("smart_person").LabelContext("<other_graph>").SaveR("<status>", "who").All()
+		`,
+		tag:    "who",
+		expect: []string{"<fred>"},
+	},
+	
+	
+}
+
+func TestGizmoMultiGraph(t *testing.T) {
```

Comment:
Maybe instead of adding a new test we can define a new `file` field in the existing `TestGizmo`?

Reply:
Challenge with that would be the current test method loads the graph upfront, then executes the tests. What I could do is add the file attribute to the test, then change the existing test method to iterate over the two test data sets, and on each iteration only the tests matching the currently graph is executed. That would allow reuse of that test method. I'll update the pull with that

Reply:
The second options might be to have a `map[string][]quad.Quad` and cache all loaded files there. This way we can use any files without affecting the way how tests are executed.

Reply:
Pull updated, take a look and see what you think, essentially similar to the map suggestion, but in two variables currently.